src/components/TransactionSimple.js

```diff
@@ -43,7 +43,7 @@ const TransactionSimple = ({
         <Container position="relative" top={4} left={4} display="inline-block">
           <P fontSize="1.6rem">{type === 'DEBIT' && ' 🎉'}</P>
         </Container>
-        <P fontSize="1rem" color="#AEB2B8">{moment(createdAt).fromNow()}</P>
```

Comment:
Maybe we should have an helper for that, because we often have this problem between our uncommon date format and moment.